### PR TITLE
akshatmittal61

### PR DESCRIPTION
**Pull request by akshatmittal61**
Added linktree for akshatmital61.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1229"></a>

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please revert the changes in the `package-lock.json` file. Let us know if you need any help! :)

**Comment by akshatmittal61** [comment]
> Please revert the changes in the `package-lock.json` file. Let us know if you need any help! :)

Hello ! I've reverted the changes as asked. Please review my PR again.
Thanks.

**Comment by akshatmittal61** [comment]
Hello !

I've reverted the changes as per asked.
Please review my PR again and let me know the status.

Thank you.

File: public/data/akshatmittal61.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      "url": "mailto:akshatmittal2506@gmail.com",
+      "icon": "email"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/akshatmittal61",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on Linkedin",
```
Comment (Panquesito7) [inline public/data/akshatmittal61.json:23]: I recommend capitalizing the `i`. What do you think?
```suggestion
      "name": "Connect with me on LinkedIn",
```

Comment (akshatmittal61) [reply public/data/akshatmittal61.json:23]: Yeah Sure. Thanks for the suggestion.

File: public/data/akshatmittal61.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      "url": "https://instagram.com/akshatmittal61",
+      "icon": "instagram"
+    },
+    {
+      "name": "Join me on your fb",
+      "url": "https://www.facebook.com/akshatmittal61/",
+      "icon": "facebook"
+    }
+  ],
+  "milestones": []
```
Comment (Pradumnasaraf) [inline public/data/akshatmittal61.json:38]:
```suggestion
  ]
```

**Review by Pradumnasaraf — APPROVED** [review]
Hey Akshat, if you don't have milestones as of now you can remove that section

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

File: public/data/akshatmittal61.json (hunk 2 above)
Comment (Pradumnasaraf) [inline public/data/akshatmittal61.json:38]:
```suggestion
  ]
```

**Review by Pradumnasaraf — COMMENTED** [review]
Hey! Akshat, a small suggestion, if you don't have any milestones as of now, you can remove that section.

File: public/data/akshatmittal61.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      "url": "https://twitter.com/akshatmittal61",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/akshatmittal61/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "See my posts on InG",
```
Comment (Shivansh-yadav13) [inline public/data/akshatmittal61.json:28]:
```suggestion
      "name": "See my posts on Instagram",
```
```suggestion
      "name": "See my posts on IG",
```
"InG" looks a bit weird 😅

**Review by emmalearnscode — APPROVED** [review]
Looks good. The empty milestones array is not necessary if it's empty but it also does no harm being there in case you want to come back and fill it up in the future. Thanks for your contribution!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by kabir0x23 — APPROVED** [review]
Looks good to me. ✔🎯

File: public/data/akshatmittal61.json (hunk 2 above)
Comment (akshatmittal61) [reply public/data/akshatmittal61.json:38]: Yeah Okay, I just thought that maybe sometime in the future I would add them, so I left the empty array. I'll delete it now though.

**Comment by akshatmittal61** [comment]
Yeah Okay, This looks better I guess.
Thanks, I'll do that now.

**Review by Pradumnasaraf — APPROVED** [review]
LGTM, thank you for the suggested changes :rocket: 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Efficient, Maintainable

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**